test/unit/src/math/Interpolant.tests.js

```diff
@@ -9,71 +9,73 @@ export default QUnit.module( 'Maths', () => {
 		// Since this is an abstract base class, we have to make it concrete in order
 		// to QUnit.test its functionality...
 
-		function Mock( parameterPositions, sampleValues, sampleSize, resultBuffer ) {
+		class Mock extends Interpolant {
 
-			Interpolant.call( this, parameterPositions, sampleValues, sampleSize, resultBuffer );
```

Comment:
`Interpolant` abstract class is not a function anymore. the test is also needed to change.